packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -110,118 +122,186 @@ describe('Facet Test Suite', () => {
           });
         });
       });
+    });
 
-      describe('when searching for a value that returns results', () => {
-        const query = 'a';
+    describe('when searching for a value that returns results', () => {
+      const query = 'a';
```

Comment:
like discussed with you before, do you think we type 2 characters will be better?

Reply:
I kept a query with a single character here to get more search results. I modified the test case that selects a search result to use a longer query though. It also validates that all facet search events are fired properly.